spec/lib/common/client/middleware/request/camelcase.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
Why do you define the camelcase middleware in both:

lib/common/client/middleware/request/camelcase.rb and
spec/lib/common/client/middleware/request/camelcase.rb

It looks like the version in /spec doesn't have any specs in it, just a redefinition of the middleware.